### ISSUE TITLE
2.4: XGBoost Model

### ISSUE THREAD

**Issue by Devarsh01** [DishaKacha7/Capstone_Group_3#2]
XGBoost Model:
1. Features Used - All features in combined dataset
2.  Feature Engineering - None
3.  Pre-processing - Feature scaling (standard scalar)

Results :
1. Accuracy - 0.7177
2.  Cohen Kappa - 0.6247
3.  Confusion Matrix - 0.67 [macro_avg of f1_score]

Areas of Improvement - Feature engineering, hyperparamter tuning
